ReactAndroid/src/main/java/com/facebook/react/uimanager/AccessibilityDelegateUtil.java

```diff
@@ -28,6 +38,17 @@
 
 public class AccessibilityDelegateUtil {
 
+  private static int sCounter = 0x3f000000;
+
+  public static final HashMap<String, Integer> sActionIdMap= new HashMap<>();
+  static {
+      sActionIdMap.put("activate", AccessibilityActionCompat.ACTION_CLICK.getId());
+      sActionIdMap.put("increment", AccessibilityActionCompat.ACTION_SCROLL_FORWARD.getId());
+      sActionIdMap.put("decrement", AccessibilityActionCompat.ACTION_SCROLL_BACKWARD.getId());      
```

Comment:
Is the idea here that these are common actions that you might want to override the label for? For example instead of saying "Double tap to [activate]" you'd set your own label on the activate action and have it say "Double tap to [custom label]"?  If so, I think AccessibilityActionCompat.ACTION_LONG_CLICK should definitely be included on this list as well.

Reply:
Sort of. The purpose of actions isn't to customize the usage hint ("double tap to...").

Android conflates the two ideas together, in that in order to add a usage hint to a component, you add a description to it's click action. And that's how usage hints are hooked up in our proposed React Native implementation of usage hints on Android.

Adding long click to the list of supported standard actions might be a good idea, but there wouldn't be any way to have that action force TalkBack to say something.

The other complication is that because of the way Android implements usage hints, all usage hints start with "double tap to...". On iOS, usage hints are simply strings, so can start with anything.

I suspect adding a "longclick" standard action name makes sense-- I'll look to see what the iOS equivalent would be.